apps/drec-ui/src/api/organization.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    OrganizationDTO,
+    useOrganizationControllerGetAll,
+    useOrganizationControllerGetOrganizationUsersByCode,
+    useOrganizationControllerGetOrganizationByCode
+} from '@energyweb/origin-drec-api-client';
+import { keyBy } from 'lodash';
+
+export const useAllOrganizations = () => {
+    const { data, isLoading: organizationsLoading } = useOrganizationControllerGetAll();
+
+    const organizations = data as OrganizationDTO[];
```

Comment:
data should have a type from the box. 
If not - something wrong in `name.controller.ts`, because type is received from auto-gen api-client package.
Also you can remove this assignment by: 
`const { data: organizations ....`

Reply:
Agreed but the reason for casting it was that otherwise, TS throws an error for keyBy and organizations.length